src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -39,6 +39,10 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
   val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ workflowStdoutStderrRoute ~ abortRoute ~
     callOutputsRoute ~ callStdoutStderrRoute ~ validateRoute ~ metadataRoute
 
+  def docsRoute = path("swagger" / "cromwell.yaml") {
```

Comment:
I'm a little confused about this `path` declaration... is this to declare the endpoint as `/swagger`?  what's the "cromwell.yaml" part for?